content/en/blog/sign-verify-plugins-rules/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "How we Sign and Verify Plugins and Rules"
+linktitle: "How we Sign and Verify Plugins and Rules"
+date: 2023-10-12
+author: Massimiliano Giovagnoli, Luca Guerra
+slug: sign-verify-plugins-rules
+images:
+  - /blog/how-we-sign-and-verify-plugins-and-rules/images/falcoctl-signature-featured.png
+---
+
+## Falco v0.36.0 and the SSC security
+
+The latest stable v0.36.0, alongside falcoctl 0.6.1 and the 0.7.0 Helm chart brought new features and new improvements in the supply chain security of Falco artifacts. The two main artifacts are plugins and rule sets as you already know. They're shipped in the OCI specification format and distributed through the official Falcosecurity OCI repositories.
```

Comment:
"new improvements in the supply chain security of Falco artifacts" -> improvements to the security of Falco's software supply chain artifacts. 

You could leave out "software" but "software supply chain" is something folks would likely search on.

Reply:
"The two main artifacts are plugins and rule sets as you already know" I tend to avoid phrases like "you already know" because if the reader doesn't already know they may stop reading or they may feel like they don't know enough about the topic or it just makes them feel bad. 

Also The two many artifacts of what?  "Falco's two main artifacts are plugins and rules.